### ISSUE TITLE
LibreOffice & OpenOffice Impress: No information reported when running slideshow

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#7703]
In LibreOffice Impress, v 5.4.2.2 (x64) or Apache OpenOffice 4.1.4, using NVDA 2017.3.  Open a slideshow, then press F5 to run.  NVDA reports: "root pane unknown FILENAME - LibreOffice Presentation  document  editable" (where FILENAME is the name of the file).

The contents of the first slide are not read, and using arrows, enter, space etc only results in the key pressed being read out (if speak typed characters and / or speak command keys enabled).  No information about the contents of the presentation is reported.  The presentation is shown on screen as expected.

**Comment by zahra21** [comment]
hello.
in libreoffice and openoffice (no matter which version i use), it seems that impress is not accessible for blind users.
i cant read the content of presentation, i think that can only use page up or page down to navigate and read the header of each slide, but cant read its content using nvda!
i sometimes use tab and impress wants me whether i want to save the changes in the file or not.
i realy dont know is impress completely inaccessible for screen reader users?
or if using it is possible, how should i read the content of presentations and each slide.

sometimes i should press f2 and read the content, but without f2, i did not find a solution to read the content of presentations.
God bless you all.

**Comment by Brian1Gaff** [comment]
Remember, anything that is fixed in issues here will not run on XP versions 
of Windows. I only mention this fact as I seem to recall your main machine 
still ids xp based.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by seanbudd** [comment]
@Qchristensen @michaelweghorn - is this still an issue?

**Comment by michaelweghorn** [comment]
> [@Qchristensen](https://github.com/Qchristensen) [@michaelweghorn](https://github.com/michaelweghorn) - is this still an issue?

Yes, it's still true in a quick test with NVDA 2025.3.2 and a current LibreOffice development version. If presenter console is enabled (which can be enabled/disabled in "Slide Show" -> "Slide Show Settings"), the number of the slide is announced, but basically nothing is announced when switching between slides otherwise.

Version: 26.2.0.0.alpha1+ (X86_64) / LibreOffice Community
Build ID: 5b067ef2767cac9a40de96cae025b1d7fb7d1ec9
CPU threads: 12; OS: Windows 11 X86_64 (build 26200); UI render: Skia/Raster; VCL: win
Locale: en-GB (en_GB); UI: en-US
Calc: threaded
